dicepy/__init__.py

```diff
@@ -15,6 +15,35 @@ class MathType(Enum):
     add = '+'
     subtract = '-'
 
+class Roll:
+    keep_type = None
+    keep_amount = 1
+    dice = None
+    value = None
+
+    def __init__(self, sides: int, dice_count: int, keep_type: KeepType = None, keep_amount: int = 1):
```

Comment:
I would default `keep_amount: int = 0` so you can check and ignore it if it's set to 0. There's other ways to do this though.

Reply:
Thinking more on this, I would suggest the following:

```python
    def __init__(self, sides: int, dice_count: int, keep_type: Optional[KeepType] = None, keep_amount: int = 1):
        self.dice = [randint(1, sides) for _ in range(dice_count)]
        self.keep_type = keep_type
        self.keep_amount = keep_amount
```

Just changing KeepType to an Optional means that you can evaluate it against None and have the type checker not be mad at you.